src/components/MultiHopTrade/components/TradeInput/components/ConfirmSummary.tsx

```diff
@@ -152,15 +152,24 @@ export const ConfirmSummary = ({
   ])
 
   const quoteHasError = useMemo(() => {
+    const tradeQuoteError = activeQuoteErrors?.[0]
+
+    // Ensures final trade quote errors are not displayed at input time for one or two render cycles as tradeQuoteSlice when reset
+    // if backing out from an errored final quote back to input
```

Comment:
Else we would end up with this flash of error when backing out to input

https://jam.dev/c/9d861b65-701a-4d72-ad59-77fcdc1fdcff